docs/core/config/dict.md

```diff
@@ -234,21 +189,23 @@ map {
 You can also access multiple SQL fields. For example
 [[setting,acl_sharing_map]] can contain:
 
-```
-map {
-  pattern = shared/shared-boxes/user/$to/$from
-  table = user_shares
-  value_field = dummy
+```[dovecot.conf]
+dict_map shared/shared-boxes/user/$to/$from {
+  sql_table = user_shares
+  value dummy {
+  }
 
-  fields {
-    from_user = $from
-    to_user = $to
+  field from_user {
+    pattern = $from
+  }
+  field to_user {
+    pattern = $to
   }
 }
 ```
 
 * The [[setting,acl_sharing_map]] always uses `1` as the value, so here the
-  `value_field` is called `dummy`.
+  `value` is called `dummy`.
```

Comment:
It's value_field again

Reply:
What do you mean exactly with "again"? Did it change between the commit that I ported from the documentation from 2024-03-25 (i.e. "b438976f5fc9a1820cedf7059939ee61f1522be9") and now?

Because then I suggest here as well to create another JIRA ticket for it. These PRs are extensive enough as they are. And if there are still undocumented changes, these should be tracked correctly.

Reply:
DOV-7009